index.html

```diff
@@ -1,220 +1,275 @@
+        <img class="card-img" src="images/snapshoot1.png" alt="Tonic"/>
+        <div class="card-content">
+          <h2 class="heading-secondary">Uber Navigation</h2>
+          <ul class="card-info">
+            <li>Uber</li>
+            <li><img src="images/Counter.svg" alt="dot" class="card-info-dot"/>
             </li>
+            <li class="input-text">Lead Developer</li>
             <li>
-                <a href="# " class="icons ">
-                    <img src="./images/engelist.png " alt="angelist ">
-                </a>
-            </li>
-        </ul>
-
-      </section>
-    </div>
-
-    <!-- second section -->
-
-    <div class="card-page">
-      <section class="work-section">
-        <div class="work-container">
-          <div class="cards-work">
-            <div class="snapshoot"><img src="./images/Snapshoot_Portfolio.png" alt=""></div>
-            <div class="primary-text">
-              <h2>Tonic</h2>
-              <img src="./images/Frame2.png" alt="">
-              <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
-            </div>
-            <div>
-              <ul class="lang">
-                <li>html</li>
-                <li>css</li>
-                <li>javaScript</li>
-              </ul>
-            </div>
-            <div class="normal-button">
-              <button type="button" class="link-buttons">See Project</button>
-            </div>
+              <img src="images/Counter.svg" alt="dot" class="card-info-dot"/></li>
+            <li class="input-text">2022</li>
+          </ul>
+          <p class="text-secondary">
+            A smart assistant to make driving more safe, efficient, and fun by
+            unlocking your most expensive computer: your car.
+          </p>
+          <ul class="card-tags">
+            <li>html</li>
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <div class="card-button" id="btn-4">
+            <button class="button" type="button">See Project</button>
           </div>
         </div>
+      </div>
+    </section>
 
-        <div class="work-container">
-          <div class="cards-work">
-            <div class="snapshoot"><img src="./images/Snapshoot_Portfolio(1).png" alt=""></div>
-            <div class="primary-text">
-              <h2>Multi-Post <br>Stories</h2>
-              <img src="./images/Frame2.png" alt="">
-              <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
-            </div>
-            <div>
-              <ul class="lang">
-                <li>html</li>
-                <li>css</li>
-                <li>javaScript</li>
-              </ul>
-            </div>
-            <div class="normal-button">
-              <button type="button" class="link-buttons">See Project</button>
-            </div>
+    <section class="about" id="about">
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_37](https://user-images.githubusercontent.com/69638013/161382648-f1927286-98d4-4e66-82c1-f0691065b451.png) |  ![Screenshot_36](https://user-images.githubusercontent.com/69638013/161382641-2e4aa445-cae8-432c-ad48-431a8e844ce1.png)

- [ ] In this section you have two blocks; The left one takes 40% which has the tag "first-part" and the right one the remaining 60%  which has the tag "skills-list" of the page's width
- [ ] Kindly ensure the  contents of each of them are aligned to the right
- [ ] kindly adjust the languages header to be just like the Figma design, It should be next to the about header 🤝 
- Just as the previous reviewer suggested try to make your design look like the original design as much as possible please add the missing icons and images  👍🏿